src/LookupTable/hooks/useGetSingleTable.hook.tsx

```diff
@@ -45,7 +45,7 @@ const useGetSingleTable = () => {
     if (structure?.name) {
       const tableName = `${GQL_TABLE_NAME_PREFIX}${capitalizeFirstLetter(
         toCamelCase(structure.name)
```

Comment:
Same.

Reply:
Thanks! I'm replacing everywhere that was using the local version of startCase